src/emulator/commandUtils.ts

```diff
@@ -251,10 +310,11 @@ async function runScript(script: string, extraEnv: Record<string, string>): Prom
 /** The action function for emulators:exec and ext:dev:emulators:exec.
  *  Starts the appropriate emulators, executes the provided script,
  *  and then exits.
- *  @param script: A script to run after starting the emualtors.
+ *  @param script: A script to run after starting the emulators.
  *  @param options: A Commander options object.
  */
 export async function emulatorExec(script: string, options: any) {
+  options = setExportOnExitOptions(options);
```

Comment:
I'd prefer any of the `options` mutation code to happen while we're still in the `commands/*` file.  Could we run this in a hook like `beforeEmulatorCommand`?

Reply:
Fixed. 

To be honest, I don't particularly like the mutation solution in general but other solutions I could think of get rather involved / dirty as well.